content/en/docs/topics/advanced.md

```diff
@@ -80,6 +80,97 @@ type PostRenderer interface {
 
 For more information on using the Go SDK, See the [Go SDK section](#go-sdk)
 
+#### Simple Example
```

Comment:
I feel like this should be at an H3 rather than an H4. H4 makes it look like it's part of the section on how to use the SDK, but this doesn't use the SDK.

Reply:
Agree, lets make it a Header 3 instead i.e. `###`